Application/Extensions/LocationFilterParametersExtensions.cs

```diff
@@ -6,7 +6,7 @@ namespace Application.Extensions;
 
 public static class LocationFilterParametersExtensions
 {
-    public static IResult<LocationFilterParameters> TryValidate(this LocationFilterParameters? parameters)
+    public static IResult<LocationFilterParameters> TryValidate(this LocationFilterParameters? parameters, bool testIsSchoolPostcode = false)
```

Comment:
Can we name this` IsValidateSchoolPostcode`?

Reply:
Yep - I've updated this to validateIsASchoolPostcode